index.d.ts

```diff
@@ -0,0 +1,2 @@
+declare const exported: any;
```

Comment:
I didn't see this typing stuff earlier, that belongs in a different CR (or just not at all, since it doesn't seem to do much atm?)

Reply:
I can throw it into another PR, but I added it because I was annoyed with the VSCode errors for missing types.

I can add another PR with actual types if that works?

Reply:
Sure.